osi_object.proto

```diff
@@ -436,6 +437,86 @@ message MovingObject
         // through.
         //
         optional double ground_clearance = 6;
+
+        // Description of each wheel.
+        //
+        // \note OSI uses singular instead of plural for repeated field names.
+        //
+        repeated WheelData wheel_data = 7;
+
+        // \brief Detailed wheel data.
+        // The focus is on the description of a wheel regarding the perceivable 
+        // information from the outside. 
+        // It is not intended to be used for e.g. dynamic calculations.
+        //
+        message WheelData
+        {
+            // The axle which contains this wheel. A value of 0 represents the
+            // front-most axle of the vehicle with higher numbers incrementing
+            // towards the rear-most axle.
+            //
+            optional uint32 axle = 1;
+
+            // The index of the wheel on the axle, counting in the direction
+            // of positive-y, i.e. right-to-left.
+            //
+            // For example, on a standard 2-axle, 4-wheel car, the rear-right
+            // wheel would be (axle=1, index=0).
+            // This concept works also for twin tires.
+            //
+            optional uint32 index = 2;
+            
+            // Position of the wheel (geomatric center of the wheel).
+            // The reference system is the vehicle frame (center of bounding box).
+            // Right-handed, x-axis shows into driving direction "forward".
+            // Rough Approximation: The z-value shows the damping.
```

Comment:
To clarify here, do you mean that the z-value should take into account things like suspension damping?  Rather than the position being fixed, e.g. while the car is at rest?

(is there a particular reason to highlight what the z-value is for?  Is it to help explain the convention?  If so, I think "x forward" from the previous line is sufficient, but don't feel strongly about it)

Reply:
Yes, I would see the damping representation in these coordinates (mostly z, but also x & y).
The line could be deleted from my side, it is just a hint, what you can do with the position.. @CCB